sky/sdk/lib/editing/input.dart

```diff
@@ -109,13 +103,23 @@ class Input extends StatefulComponent {
 
     return new Listener(
       child: input,
-      onPointerDown: (_) => keyboard.showByRequest()
+      onPointerDown: focus
     );
   }
 
+  void focus(_) {
+    if (Focus.at(this)) {
```

Comment:
I'll (or someone'll) be doing nested focus in another CL. We can change it then if at() no longer makes sense.